R/ggThemeAssist.R

```diff
@@ -509,9 +520,15 @@ ggThemeAssist <- function(){
 
         result <- formatR::tidy_source(text = result, output = FALSE, width.cutoff = 40)$text.tidy
         result <- paste(result, collapse = "\n")
-        rstudioapi::insertText(result)
+        if (addin) {
+          rstudioapi::insertText(result)
+        }
+      }
+      if (addin) {
+        invisible(stopApp())
+      } else {
```

Comment:
Sorry, I didn't have time to try it myself, but I guess this else should be placed in line 525?

Otherwise I'm happy to merge your PR, one commit would be preferred but isn't a must. If you want to, take a look at http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git. 

Thanks for your support!